### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by khthana**
test

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature-1 branch
+
+One more change in feature-1 branch
```
Comment (khthana) [inline feature1/one-more-file.txt:3]: test

Comment (thana-ho) [reply feature1/one-more-file.txt:3]: Thank you.

File: new-feature.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This file is created in feature-1 branch
```
Comment (thana-ho) [inline new-feature.txt:1]: Good

**Review by thana-ho — APPROVED** [review]

**Review by khthana — COMMENTED** [review]
Good

**Pull request merged**